### DISCUSSION TITLE
Mac: “SQLiteStudio-3.4.4-osx-installer” cannot be opened because the developer cannot be verified.

### DISCUSSION THREAD

**Discussion by chuckcoleman** [pawelsalawa/sqlitestudio#4889] [Q&A]
### Details
Error is in subject line.

### Steps to reproduce
Download Mac 3.4.4 image, then click on "SQLiteStudio-3.4.4-osx-installer"

### Operating system
Mac OSX 13.6 (22G120)

### SQLiteStudio version
3.4.4

**Comment by pawelsalawa** [comment]
See #4451
